includes/class-migration.php

```diff
@@ -207,6 +214,38 @@ public static function async_migration( $version_from_db ) {
 		}
 	}
 
+	/**
+	 * Asynchronously runs upgrade routines.
+	 *
+	 * @param string $callback The callback to run.
+	 * @param mixed  ...$args  The arguments to pass to the callback.
+	 */
+	public static function async_upgrade( $callback, ...$args ) {
```

Comment:
```suggestion
	public static function async_upgrade( $callback ) {
```

Should we get rid of this now that we're calling `func_get_args()` below?

Reply:
yeah, I wonder if func_get_args is even needed…

Reply:
Might not be with the new structure. But this works :)